docs/source/models.rst

```diff
@@ -1902,6 +1914,25 @@ argument. The following values are supported:
     df = spark_df.withColumn("prediction", pyfunc_udf(struct("name", "age")))
 
 
+If you want to use conda to restore the python environment that was used to train the model,
+set the `env_manager` argument when calling :py:func:`mlflow.pyfunc.spark_udf`.
+
+
+.. rubric:: Example
+
+.. code-block:: py
+
+    from pyspark.sql.types import ArrayType, FloatType
+    from pyspark.sql.functions import struct
+
+    pyfunc_udf = mlflow.pyfunc.spark_udf(
+        "path/to/model", result_type=ArrayType(FloatType()),
+        env_manager="conda"  # Use conda to restore the environment used in training
+    )
```

Comment:
Is this line correct? The first argument of `spark_udf` is

Reply:
We're missing the `spark` argument, which is the first argument to the function. See https://github.com/mlflow/mlflow/pull/5677/files#diff-27092883ef9a4a81ab9b5ea59442ab48dd314babd61428270948fcfae0a80f28R533-R541.

Reply:
The example above this one also has the same issue. Let's fix it.

Reply:
Can we add `import mlflow` so we can copy the code in a script and run it?